_updates/2020-04-08-april.md

```diff
@@ -0,0 +1,40 @@
+---
+title: April 2020
+---
+
+## Work From Home 🏠
+
+Because of the pandemic we are now working from home. This doesn't mean that the developement stopped. Here are the changes we made.
```

Comment:
```suggestion
Because of the pandemic we are now working from home. This doesn't mean that the development is stopped. Here are the changes we made.
```